src/content/en/fundamentals/performance/http2/index.md

```diff
@@ -254,7 +254,7 @@ reassemble them on the other end.
 ![HTTP/2 request and response multiplexing within a shared connection](images/multiplexing01.svg)
 
 The snapshot captures multiple streams in flight within the same connection. The
-client is transmitting a DATA frame (stream 5) to the server, while the server
+client is transmitting a `DATA` frame (stream 5) to the server, while the server
```

Comment:
I've re-read the paragraph and I was wrong. I'll corrent and commit again. Thank you for your feedback.